portfolio/src/main/webapp/index.html

```diff
@@ -2,15 +2,68 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+      <h1>Desmond Michael Talton</h1>
+
+            <h3>
```

Comment:
Is h3 used for styling purpose? If yes, consider using css to style it.
Look into "semantic html":
Semantic HTML is the use of HTML markup to reinforce the semantics, or meaning, of the information in webpages and web applications rather than merely to define its presentation or look.

Reply:
Thanks. I'll look into that.
h3 is for styling. I intend to brush over this entire project with CSS after I have everything down and in the places I want them to be (or something close to it at least). 

Also, is it okay if I attach a screenshot of the portfolio page after the next pull request (which will come in maybe 2 hours) since I have made a good amount of changes since the last commit you've gotten to see?

Reply:
How about simply removing the `<h3> <h2> ` introduced for styling purpose, just left them unstyled until later phase.